### PR TITLE
Purge "will" from Quick Start Guides

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Plus the occasional drive-by.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/quickstart-public-cloud/pages/clients.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ Most new features and enhancements are only available for Salt-managed clients,
 For more details, see xref:client-configuration:supported-features.adoc[]
 
 In the {productname} {webui}, navigate to menu:Systems[Bootstrapping], then fill in the ``Host``, ``SSH Port``, ``User`` and ``Password`` fields.
-Make sure you use stable FQDNs for the ``Host`` field, or {productname} will not be able to find your host when your Public Cloud gives you a different short-lived FQDNS.
+Make sure you use stable FQDNs for the ``Host`` field, or {productname} can not find your host when your Public Cloud gives you a different short-lived FQDNS.
```
Comment (keichwa) [inline modules/quickstart-public-cloud/pages/clients.adoc:11]:
```suggestion
Make sure you use stable FQDNs for the ``Host`` field, or {productname} cannot find your host when your Public Cloud gives you a different short-lived FQDNS.
```

File: modules/quickstart-sap/pages/prepare-clients.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 Your SAP cluster requires several client systems.
 Prepare your clients on physical or virtual hardware, and ensure you have {sles}{nbsp}15 for SAP installation media ready.
-You will not be able to create an SAP cluster without the {sles} SAP extension, as it provides tooling specific to SAP.
+You cannot create an SAP cluster without the {sles} SAP extension, as it provides tooling specific to SAP.
```
Comment (keichwa) [inline modules/quickstart-sap/pages/prepare-clients.adoc:6]: yes, you can!

**Review by keichwa — APPROVED** [review]

File: modules/quickstart-public-cloud/pages/clients.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/quickstart-public-cloud/pages/clients.adoc:11]: Clearly my brain just refuses to learn this :woman_facepalming:

Comment (Loquacity) [inline modules/quickstart-public-cloud/pages/clients.adoc:11]:
```suggestion
Make sure you use stable FQDNs for the ``Host`` field, or {productname} can not find your host when your public cloud gives you a different short-lived FQDNS.
```

File: modules/quickstart-public-cloud/pages/clients.adoc [hunk 3]
```diff
@@ -38,7 +38,7 @@ Do not do this for production systems.
 ====
 
 
-When the bootstrap process has completed successfully, your client will be listed at menu:Systems[System List].
+When the bootstrap process has completed successfully, your client is listed at menu:Systems[System List].
 
 * If you want more control over the process, have to register many clients, or are registering traditional clients, we recommend that you create a bootstrap script.
```
Comment (Loquacity) [inline modules/quickstart-public-cloud/pages/clients.adoc:43]:
```suggestion
* If you want more control over the process, have to register many clients, or are registering traditional clients, create a bootstrap script.
```

File: modules/quickstart-sap/pages/prepare-clients.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/quickstart-sap/pages/prepare-clients.adoc:6]: :sweat_smile:

**Pull request merged**